demos/init-app.php

```diff
@@ -29,6 +29,12 @@
     'catchExceptions' => (bool) ($_GET['APP_CATCH_EXCEPTIONS'] ?? true),
     'alwaysRun' => (bool) ($_GET['APP_ALWAYS_RUN'] ?? true),
 ]);
+unset($_SERVER);
+unset($_GET);
+unset($_POST);
+unset($_FILES);
+unset($_COOKIE);
```

Comment:
@abbadon1334 one last thing to address

1. $_COOKIE should have same API as $_GET, ... have
2. this unset is breaking one Behat test

Reply:
Am i wrong, or this is needed only for PHPUnit for continuous/fast calls without isolation of requests?

Reply:
The `unset($_XXX)` there is for non-phpunit actually, it is there to make sure every atk4/ui component use the request details from App/request object. See the Behat failure, it can be easily reproduced localy /w any browser, if these unsets are present, the filtering does not work.

Reply:
please make the tests passing - but please do not force the branch or merge in develop - I will then do final review and merge